react/features/invite/functions.js

```diff
@@ -616,3 +616,23 @@ export function _decodeRoomURI(url: string) {
 
     return roomUrl;
 }
+
+/**
+ * Returns the stored conference id.
+ *
+ * @param {Object} state - The state of the app.
+ * @returns {string}
+ */
+export function getConferenceId(state: Object) {
```

Comment:
We usually use a `stateful: Object | Function` instance here, so then both `state` and `getState` can be passed to these state dependant functions. Can you please adjust (same for some more below).

Se example here: https://github.com/jitsi/jitsi-meet/blob/eed5e31ec9c78d6bc317c27412036e73a974ac9e/react/features/base/participants/functions.js#L79